library/core/src/num/int_macros.rs

```diff
@@ -1165,7 +1204,7 @@ macro_rules! int_impl {
         #[rustc_const_stable(feature = "const_checked_int_methods", since = "1.47.0")]
         #[must_use = "this returns the result of the operation, \
                       without modifying the original"]
-        #[inline]
```

Comment:
Why did you change this? I don't think we need to adjust the inline attribute on this and `checked_shr`, or at least we don't anymore. I know the codebase has shifted a bit over the course of this PR.

Reply:
Yeah, I can just remove this.